src/main/java/co/omise/models/DisputeReasonCode.java

```diff
@@ -0,0 +1,22 @@
+package co.omise.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum DisputeReasonCode {
+    @JsonProperty("cancelled_recurring_transaction")cancelledRecurringTransaction,
```

Comment:
This work like this out of the box? Don't we need to add custom logic to the serializer?

Reply:
@Namiii In my opinion, I think we need it. That makes it easy to use but after re-think, should we just let it be String and let Customer map it by themselves? 🤔

Reply:
No if it works fine, we should leave it like this. I was just wondering

Reply:
Got it. 😄 👌